modules/docs/src/Volo.Docs.Web/Pages/Documents/Project/Index.cshtml.cs

```diff
@@ -36,6 +34,10 @@ public class IndexModel : AbpPageModel
         public string LanguageCode { get; set; }
 
         public bool DocumentFound { get; set; } = true;
+        
+        public bool EntityFound { get; set; } = true;
```

Comment:
It should be renamed to `ProjectFound`.

`_projectAppService.GetAsync(ProjectName)`